### ISSUE TITLE
JupyterLab 1.0 not supported

### ISSUE THREAD

**Issue by vilim** [JuliaGizmos/WebIO.jl#320]
Trying to install the lab extension on JupyterLab 1.0.2 raises:

```
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=1.0.1 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/application
>=1.0.1 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/docmanager
>=1.0.1 <2.0.0          >=0.19.2 <0.20.0@jupyterlab/notebook
>=4.0.1 <5.0.0          >=3.2.1 <4.0.0 @jupyterlab/services
```

Downgrading JupyterLab to 0.35 works.

**Comment by crstnbr** [comment]
Can confirm this.

**Comment by travigd** [comment]
I'm aware of this. It shouldn't be too hard to fix (hopefully maybe).

**Comment by travigd** [comment]
This will be fixed by #319.

**Comment by vilim** [comment]
Installation works now, however I get the WebIO not detected error (after uninstalling everything, running jupyter lab clean, and running WebIO.install_jupyter_serverextension and  WebIO.install_jupyter_labextension and verifying the jupyter location is correct). There are no errors in the JavaScript browser console, and jupyter labextension list shows the extension as installed on the correct version (0.8.8)
